### ISSUE TITLE
Relocate tier 1 description strings?

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/models-of-the-hydrogen-atom#133]
For code review https://github.com/phetsims/models-of-the-hydrogen-atom/issues/125:

@AgustinVallejo set me the following Slack#DM:

> A note about A11y Tier 1: In QM, we're adding all the accessibleName and accessibleHelpText under `QMStrings.a11y.translatable.etc`.... I realize the a11y strings in MOTHA are not under the translatable nesting. Do you think the translatable thing is something new, or MOTHA won't have that? It was JG who told me to put them under `translatable`.

**Comment by pixelzoom** [comment]
I'm not aware of what @AgustinVallejo is referring to.  @jessegreenberg @arouinfar please clarify.  Do we need to relocate the `accessibleName` and `accessibleHelpText` string in models-of-the-hydrogen-atom-strings_en.json?

**Comment by pixelzoom** [comment]
In Slack#developer, @jessegreenberg posted a PSA about `a11y.translatable`.  There were many questions, including:

1. Why are we relying on a parent node (like `a11y.translatable`), and not using metadata, to determine which strings are translatable?
2. Can we get rid of the a11y key?
3. What is an example of strings that should not be translatable but are under `a11y`?

Given these questions, should I incur the expense of moving these strings in models-of-the-hydrogen-atom-strings_en.json, regenerating the String.ts, and then adjusting many source files? (The last step is the most expensive.)

**Comment by pixelzoom** [comment]
> Given these questions, should I incur the expense of moving these strings in models-of-the-hydrogen-atom-strings_en.json, regenerating the String.ts, and then adjusting many source files?

I didn't know who to ask about this, so I went ahead and did it.  If PhET decides to change the `a11y.translatable` convention after MOTHA is published, it will require an MR.

**Comment by pixelzoom** [comment]
babel strings don't looks like what I expected after this change. In Slack#dev-public, I asked:

Related to `a11y.translatable`… After relocating all of MOTHA’s description strings to be under `a11y.translatable`, and doing a grunt modulify, I still see this in `babel/_generated_development_strings_/models-of-the-hydrogen-atom_all.json`:

```
    "a11y": {} 
```

I expected to see:

```
    "a11y": {
      "translatable": {
         ... lots of strings here
      }
    }
```
Please clarify.

**Comment by jessegreenberg** [comment]
Thanks @pixelzoom, that sounds good. We are going to discuss this organization further at the next dev meeting. If we can avoid an MR that is a bonus but (IMO) decisions should not block this release.

**Comment by pixelzoom** [comment]
Thanks @jessegreenberg. 

So nothing else to do here, closing.

**Comment by jessegreenberg** [comment]
After discussion, we decided to use metadata instead of an `a11y.translatable` (https://github.com/phetsims/rosetta/issues/449#issuecomment-2689131020). So https://github.com/phetsims/models-of-the-hydrogen-atom/commit/f41d43b398fbe7e9a2bfca53fce5a4885db65734 should be reverted. Sorry for the extra work here.

Feel free to assign to me if you would like me to do it.

**Comment by pixelzoom** [comment]
Based on https://github.com/phetsims/rosetta/issues/449#issuecomment-2689131020, it looks like we are using `"translatable: false`.   But that comment also says that Rosetta needs to be updated to recognize the `translatable` metadata.  Is Rosetta going to be updated before MOTHA 1.0 release?  If not,  then isn't that going to be a problem, because all of these strings will be translatable?

**Comment by jessegreenberg** [comment]
We are not going to use the `translatable` key but we are still using the `a11y` key for now. Nothing under the `a11y` key will be translatable.

**Comment by pixelzoom** [comment]
Sorry, I was confused, probably by the pointer to https://github.com/phetsims/rosetta/issues/449#issuecomment-2689131020.  

So I should just revert https://github.com/phetsims/models-of-the-hydrogen-atom/commit/f41d43b398fbe7e9a2bfca53fce5a4885db65734, so that these strings are under "a11y" instead of "a11y.translatable".  Is that correct?

**Comment by jessegreenberg** [comment]
Yes, that is correct.

**Comment by pixelzoom** [comment]
I couldn't simply revert https://github.com/phetsims/models-of-the-hydrogen-atom/commit/f41d43b398fbe7e9a2bfca53fce5a4885db65734, because there have been other changes to the strings file since that commit.  But I manually undid the work, and pushed https://github.com/phetsims/models-of-the-hydrogen-atom/commit/081930f43d62fc90853865bba31689e2ab455f95.

Closing.
